llvm/lib/TargetParser/AArch64TargetParser.cpp

```diff
@@ -349,6 +349,14 @@ void AArch64::ExtensionSet::reconstructFromParsedFeatures(
   }
 }
 
+uint64_t AArch64::ExtensionSet::toCpuSupportsMask() const {
```

Comment:
Hmm, I am thinking we may want to reuse this once we change [getFMVPriority](https://github.com/llvm/llvm-project/pull/119231/files#diff-d380d584ecd28d0608428cfaf49b7d7a8df5ec765f7d8e94ab95f65bf7534cfcR51) (it is under discussion). Perhaps we want a better naming, like `toFMVFeatureMask`? I used the naming convetion inspired from `toLLVMFeatureList` in the same class.

Reply:
I guess you could always move the functionality back into a separate function if future changes require that. That said, I would prefer `getFMVFeatureMask` if you keep it, because it makes it clear that the bitmask describes features for the purpose of FMV.